django/contrib/admin/static/admin/css/base.css

```
@@ -56,6 +56,9 @@ html[data-theme="light"],
     --object-tools-bg: var(--close-button-bg);
     --object-tools-hover-bg: var(--close-button-hover-bg);
 
+    --blockquote-color: #484848;
+    --blockquote-border: var(--blockquote-color);
```

```suggestion
    --blockquote-color: #484848;
```
As you are defining that the border is always the same color, can we just define the one color?